plutus-scb-client/src/Types.purs

```diff
@@ -34,6 +34,7 @@ newtype State
   = State
   { fullReport :: WebData (FullReport ContractExe)
   , chainState :: Chain.State
+  , contractSignatures :: Map ContractInstanceId (WebData (ContractSignatureResponse ContractExe))
```

Comment:
seems like a strange type, saving the last ajax call state for every contract instance? I would expect the `WebData` to be dealt with elsewhere?

Reply:
Fairly soon you'll be able to see a list of contracts and call an endpoint on one of them. At that point, that single contract will be `(re)Loading` while all the others stay as they are.